### ISSUE TITLE
Block option for any copy tools

### ISSUE THREAD

**Issue by corsirhino** [mcneel/rhinoissues#39]
I would like to be able to create a block using the Copy/array tools.
This could be very usefull for large interior design where I have to create lot of chairs and table copy.
Thanks

**Comment by BobMcNeel** [comment]
I'm not sure I understand.

Would you like an option on the Copy and Array commands to make a "Block"? What about other commands that have a Copy option?

How is this much different than just running the Block command first?

It seems like adding a Block option all the commands that have a Copy option would just add unnecessary complexity.

Am I confused?

**Comment by corsirhino** [comment]
YES! an option on the Copy and Array to make blocks. Other tools can have too.
The difference it's in the interaction. Architects, or maybe just the one I know, are lazier than the brown dog.
How many time I saw 300Mb file just because the chair and table where not copied as a block.
It's like to have Copy or Instance.

I still creating my own script to do this... but why not have it as option?

**Comment by BobMcNeel** [comment]
Ok. It seems that I do understand your request.  

Unfortunately, I think it would be a huge mistake to add this complexity to every command that has a "Copy" option.

First, I don't think it remove any "clicks" in most cases. S

econd, I don't think users would be any more likely to use the options when they should. In fact, I expect they would start using it when they should not. That is, they would be creating blocks inside of blocks, endlessly nested without knowing it.
